test/unit/xml_description_test.py

```diff
@@ -3,22 +3,42 @@
 
 from .test_helper import *
 
-from kiwi.exceptions import (
-    KiwiSchemaImportError,
-    KiwiValidationError,
-    KiwiDescriptionInvalid,
-    KiwiDataStructureError
-)
+from kiwi.exceptions import *
```

Comment:
we "allow" the wildcard import only in the tests, wouldn't that be fair enough ?
exceptions not handled in a test will result in a coverage error, thus I thought it is ok to have a wildcard import for exceptions in tests